code/string_algorithms/naive_pattern_search/naive_pattern_search.cpp

```diff
@@ -0,0 +1,39 @@
+// part of cosmos by opengenus foundation
+
+#include <iostream>
+#include <vector>
+using namespace std;
+
+vector<int> naive_search(string pattern, string text)
```

Comment:
could we return pair it consists of bool and vector or pair it consists of bool and iterator?
e.g., std::pair<bool, std::string::iterator> -> the first is true if found, the second is text start at.